app/presenters/hyrax/work_show_presenter.rb

```diff
@@ -38,7 +39,7 @@ def page_title
     delegate :title, :date_created, :description,
              :creator, :contributor, :subject, :publisher, :language, :embargo_release_date,
              :lease_expiration_date, :license, :source, :rights_statement, :thumbnail_id, :representative_id,
-             :member_of_collection_ids, to: :solr_document
+             :rendering_ids, :member_of_collection_ids, to: :solr_document
```

Comment:
Should this move to `Hyrax::ManifestComponents` where it is used?

Reply:
Yes, done.